lib/commons/aria/get-element-unallowed-roles.js

```diff
@@ -59,50 +56,32 @@ function getRoleSegments(vNode) {
 function getElementUnallowedRoles(node, allowImplicit = true) {
   const vNode =
     node instanceof AbstractVirtuaNode ? node : getNodeFromTree(node);
-  const { nodeName } = vNode.props;
-
   // by pass custom elements
   if (!isHtmlElement(vNode)) {
     return [];
   }
-
+  // allow landmark roles to use their implicit role inside another landmark
+  // @see https://github.com/dequelabs/axe-core/pull/3142
+  const { nodeName } = vNode.props;
+  const implicitRole = getImplicitRole(vNode) || landmarkRoles[nodeName];
```

Comment:
is nodeName always lower case (XHTML)?

Reply:
It is. We cast nodeName to lower case before assigning it to vNode.props. It's more consistent that way.